src/api/controllers/IdentityController.js

```diff
@@ -284,6 +285,53 @@ class IdentityController extends PointSDKController {
 
         return this._response({eligibility, reason});
     }
+
+    async resolveDomain() {
+        const SUPPORTED_NAME_SERVICES = ['sns', 'ens'];
+        const SUPPORTED_TLD = ['.sol', '.eth'];
```

Comment:
I was in doubt whether to hard-code these here or pass them in the _config_. I landed on this option thinking that these will rarely change and we probably won't need the flexibility of having different values in different environments. Also, _config_ is quite crowded as it is.

But if you think it's better to have them in _config_, please let me know.

Reply:
It's fine to have it hard coded for now.

The thing I don't like is the service name that we expect to be passed. I think we should not rely on client code, and determine the service from the TLD itself instead, since there is no other way to use the services except requesting `sns` for `.sol` and `ens` for `.eth` domains.

Reply:
Yeah, I wasn't sure about this. In theory, there could be other TLDs, and potentially some conflicts. For example, ethereum has `.test` in Ropsten. Solana could create a "domain program" with `.test` too.

But I guess the probability of collision is too small, and we are mostly interested in supporting `.sol` and `.eth`, so I'll make this change.